src/app/multiple-records/multiple-records.component.html

```diff
@@ -0,0 +1,88 @@
+<div class="govuk-width-container">
+  <a [routerLink]="['/']" class="govuk-back-link">Back</a>
+  <main class="govuk-main-wrapper">
+    <div class="govuk-grid-row govuk-!-margin-bottom-3">
+      <div class="govuk-grid-column-full">
+        <h1 class="govuk-heading-xl govuk-!-margin-bottom-5">
+          Select a technical record
+        </h1>
+        <p class="govuk-body govuk-!-font-size-19">
+          Multiple technical records found matching this search
+        </p>
+      </div>
+    </div>
+
+    <ng-container *ngFor="let techRecord of techRecords | FilterMultipleRecords; let i = index">
```

Comment:
we need to preempt this filter and move them to a state. so that we can write a selector use it in the container. Lets discuss